Sets the FEATURES variable with good default options.

Later patches can then add/remove specific features to/from it.

Index: etc/make.conf
===================================================================
--- etc.orig/make.conf
+++ etc/make.conf
@@ -283,7 +283,7 @@ PORT_LOGDIR="/var/log/ebuilds"
 #     specified on every run. Useful options include --ask, --verbose,
 #     --usepkg and many others. Options that are not useful, such as --help,
 #     are not filtered.
-EMERGE_DEFAULT_OPTS="--nospinner"
+EMERGE_DEFAULT_OPTS="--nospinner --with-bdeps=y"
 #
 # INSTALL_MASK allows certain files to not be installed into your file system.
 #     This is useful when you wish to filter out a certain set of files from
@@ -322,6 +322,16 @@ PORTAGE_NICENESS="10"
 # splitdebug test userpriv usersandbox"
 # FEATURES sorted alphabetically for easier maintenance.
 FEATURES="
+ ccache
+ metadata-transfer
+ notitles
+ -parallel-fetch
+ sandbox
+ sfperms
+ unmerge-orphans
+ userfetch
+ userpriv
+ usersandbox
 "
 
 # CCACHE_SIZE and CCACHE_DIR are used to control the behavior of ccache, and
